### ISSUE TITLE
Readme is hard to follow

### ISSUE THREAD

**Issue by hasenbalg** [danielwelch/hassio-zigbee2mqtt#252]
**zigbee2mqtt add-on version (if edge, please report commit hash):**
1.5.1
**Operating environment (HassOS, Virtual Machine, Device/platform running Hass.io):**

raspberry 3

**Description of problem:**
Hi, I'm very new to all the home automation things.
I'm not able to follow the instructions given in the read-me file to setup. Eg. I cannot find the setting to pair my Zigbee devices.

Are there any dependencies to other hassio extensions? Do they install along  hassio-zigbee2mqtt?
Is it be possible to clarify the instruction?

Thank you!

**Your entire configuration from the frontend (with sensitive fields redacted)**:
```json
// replace this line with your pasted configuration
```

**Your logs from Hass.io**

**Comment by danielwelch** [comment]
Hey, thanks for posting.

> Are there any dependencies to other hassio extensions? Do they install along hassio-zigbee2mqtt?

No, there are no external dependencies. The README probably assumes some basic knowledge about Hassio add-ons. I recommend reading the Home Assistant docs regarding Hassio and add-ons. 

> I cannot find the setting to pair my Zigbee devices

If you mean the setting to allow for device pairing, it’s the permit_join option. There are a number of ways to allow pairing. We recommend setting permit_join to false for security reasons, but when you’re setting things up, you can just set this to true to allow pairing. The alternative is to allow pairing via an MQTT message sent to zigbee2mqtt. You can send this message however you want (Home Assistant service call, command line, third party MQTT broker web/web client, etc.). The README just provides one suggestion.

If you’re just wondering how to pair devices in general, I’d advise reading the zigbee2mqtt docs, and seeking help in the zigbee2mqtt repository.

**Comment by aurelijusrozenas** [comment]
Hi,

I also have trouble trying to setup this :/
I guess it's common knowledge for devs but it maybe how other people feel when they starting out with HA and zigbee2mqtt.

Some information is here, some - in the [official zigbee2mqtt docs](https://www.zigbee2mqtt.io/integration/home_assistant.html) but they say use readme from this repo.
So I installed addon, enabled permit_join, paired light bulb and... now what? Is this the end for configuration or should I follow official docs to setup `MQTT discovery` and so on? Sorry it's just not clear.
Home assistant does not show anything (should it be in Devices menu)? I see device is paired in the mqtt log but I don't see any way to use it in HA.
I also enabled `MQTT discovery` as described in zigbee2mqtt docs but this did not help to see devices.
I swear I had this working last year but now the knowledge is lost to time...

**Comment by danielwelch** [comment]
@aurelijusrozenas basically all of your questions relate to zigbee2mqtt itself, not the add-on. You should follow the zigbee2mqtt documentation, with the only differences being the installation process and the configuration method and associated subtle differences outlined in the README. If everything has installed, started, and zigbee2mqtt is running, it’s likely not an add-on problem but some problem with your zigbee2mqtt config (with the most common exception being networking issues specific to Hassio / Docker).

**Comment by aurelijusrozenas** [comment]
@danielwelch thanks for the explanation, it helped to wrap head around HA, hassio, zigbee2mqtt and add-ons :)

**Comment by aurelijusrozenas** [comment]
@hasenbalg additionally you can install Zigbee2MqttAssistant add-on. There is option to permit join and it makes easier to debug zigbee network.
